services/src/tasks/mod.rs

```diff
@@ -67,6 +67,10 @@ pub trait Task<C: TaskContext>: Send + Sync {
         None
     }
 
+    fn task_description(&self) -> Option<String> {
+        None
```

Comment:
[43c32ab](https://github.com/geo-engine/geoengine/pull/820/commits/43c32abcf5fc9195b699e0ebdb43861f5feddfb0)

I am not sure if it would be better to make task_description async or use block_on. In the normal case it is not required to make the method async.

Reply:
[8a9445d](https://github.com/geo-engine/geoengine/pull/820/commits/8a9445dd3891a3baec33647cd772a8c350df0dd3)

I made task_description async because the tests were failing.

Reply:
[d052c3b](https://github.com/geo-engine/geoengine/pull/820/commits/d052c3b88574c72dd7eda86fd6063ad8d0f0dbc0)